src/Catalyst.Core.Lib/DAO/TransactionBroadcastDao.cs

```diff
@@ -21,35 +21,22 @@
 
 #endregion
 
-using System;
-using System.Collections.Generic;
 using AutoMapper;
 using Catalyst.Abstractions.DAO;
+using Catalyst.Core.Lib.DAO.Transaction;
 using Catalyst.Protocol.Wire;
-using Google.Protobuf.WellKnownTypes;
 using MongoDB.Bson.Serialization.Attributes;
 
 namespace Catalyst.Core.Lib.DAO
 {
+    /// <summary>
+    /// @TODO we shouldnt be saving TransactionBroadcast, this is a wire only object,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4600656)